DatWeatherDoe/UI/Decorator/Condition/WeatherCondition.swift

```diff
@@ -30,7 +30,12 @@ enum WeatherCondition {
     case partlyCloudyRain
 
     case thunderstorm
-    
+
+    // Used as decoration in the dropdown and not for weather conditions
+    case location
```

Comment:
These should probably not be in weather condition and have their own enum or equivalent

Reply:
Will do.